File: practice-challenges/replace-vowels/replace_vowels.py
"""Python solution to Replace Vowels"""

def replace_vowels(chars):
    """Given list of chars, return a list with vowels replaced by '*'."""

    for i, char in enumerate(chars):
        if char in "AEIOUaeiou":
            chars[i] = "*"
    
    return chars


# Test with sample input
print(replace_vowels(['h', 'i']))
print(replace_vowels(['o', 'o', 'o']))
print(replace_vowels(['z', 'z', 'z']))

File: practice-challenges/replace-vowels/replaceVowels.js
// Solution to Replace Vowels

function replaceVowels(chars) {
    // Given list of chars, return a list with vowels replaced by '*'.

    for (let i=0; i < chars.length; i += 1) {
        if ("AEIOUaeiou".includes(chars[i])) {
            chars[i] = "*";
        }
    }
    
    return chars
}

// Test with sample input
console.log(replaceVowels(['h', 'i']))
console.log(replaceVowels(['o', 'o', 'o']))
console.log(replaceVowels(['z', 'z', 'z']))
